google/auth/transport/grpc.py

```diff
@@ -51,15 +52,19 @@ class AuthMetadataPlugin(grpc.AuthMetadataPlugin):
             add to requests.
         request (google.auth.transport.Request): A HTTP transport request
             object used to refresh credentials as needed.
+        audience (Optional[str]): An audience like "pubsub.googleapis.com".
```

Comment:
Here audience is not prefixed with `https://`. The [AIP](https://google.aip.dev/auth/4111) specifies audience as a full URI with `https://`. Could we make this consistent, or give it a different name, e.g. `service_address`?

The proto refers to it as [`default_host`](https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto#L41)

Reply:
👍 Renamed to `default_host`.